src/api/Annotations.js

```diff
@@ -1,7 +1,18 @@
 // @flow
 import merge from 'lodash/merge';
+import type { BoxAnnotationPermission } from '../common/types/feed';
```

Comment:
Should this be located below with the other type imports?

Reply:
😱